### PR TITLE
docs updates

### PR DESCRIPTION
**Pull request by rcanfield2**

### CODE REVIEW

**Comment by rcanfield2** [comment]
Issue #296

File: aopy/analysis/base.py [hunk 1]
```diff
@@ -783,7 +796,7 @@ def get_sgram_multitaper(data, fs, win_t, step_t, nw=None, bw=None, adaptive=Fal
         tuple: Tuple containing:
             | **fxx (np.array):** spectrogram frequency array (equal in length to win_t * fs // 2 + 1)
             | **txx (np.array):** spectrogram time array (equal in length to (len(data)/fs - win_t)/step_t)
-            | **Sxx (len(fxx):** x len(txx) x nch): multitaper spectrogram estimate. Last dimension squeezed for 1-d inputs.
+            | **Sxx (len(fxx): x len(txx) x nch):** multitaper spectrogram estimate. Last dimension squeezed for 1-d inputs.
```
Comment (leoscholl) [inline aopy/analysis/base.py:799]: I think the colon after len(fxx) will break things

File: aopy/precondition/base.py [hunk 2]
```diff
@@ -613,7 +613,7 @@ def calc_ks_waveforms(raw_data, sample_rate, spike_times_unit, templates, channe
         time_before (float, optional): time (us) to include before the start of each trial
         time_after (float, optional): time (us) to include after the start of each trial
     
-    returns
+    returns:
         tuple: tuple containing:
```
Comment (leoscholl) [inline aopy/precondition/base.py:617]: Does the r in Returns need to be capitalized?

**Review by leoscholl — APPROVED** [review]
Very helpful, thanks!

**Pull request merged**